feat: ✨ add concluding remarks (#105)
rostools/github-intro: sessions/conclusion.qmd

# Concluding remarks {#sec-conclusion}

<!-- TODO: Add content that mirrors/reflects on the bigger picture from the intro (once that has been made) -->

> We've reached the end of the workshop! Let's take a look at what we've
> covered.
> 
> We've now covered the basics of GitHub and Git, working with files on
> GitHub, and making and using GitHub Issues.

Reviewer: ```suggestion
We've reached the end of the workshop! To summarise, we've covered the 
basics of GitHub and Git, working with files on GitHub, and making and 
using GitHub Issues.
```

More specifically, we've:

-   Created a repository on GitHub
-   Worked with files in our own repository on GitHub: Created, edited,
    renamed, and deleted recipe files and folders
-   Created and commented on GitHub issues

> -   Added and removed collaborators
> -   Worked with files in our collaborator's repository on GitHub

Reviewer: ```suggestion
```

Let's spend a few minutes thinking about how you might use GitHub in
your own work.

## :speech_balloon: Discussion activity: Consider how you'd use GitHub in your work

> **Time: \~4 minutes.**

Reviewer: ```suggestion
**Time: \~6 minutes.**
```

Learning is most effective when there is some connection to what you do
in your daily work life. On your own and based on what we've covered so
far, think about how you might use GitHub in your own work. Then, share
some of your ideas with your neighbour.

1.  Take \~1 minute to think how you might use GitHub in your own work.
2.  Then, pair up with your neighbour and take 3 minutes (1.5 minutes
    each) to share your ideas.
3.  After this, we'll come back together and share some of the
    ideas---we might inspire each other!

## What's next?

We hope you've enjoyed this workshop and that you've learned something

> new about GitHub and Git. If you want to learn more about Git and
> GitHub, we are working on a series of workshops that will cover more
> advanced topics:
> 
> 1.  Work-in-progress:
>     [quarto-website-intro](https://github.com/rostools/quarto-website-intro) -
>     Create a website using GitHub and Quarto.
> 2.  Work-in-progress: [git4cats]() - Work in
>     **C**oll**A**borative**T**eam**S** using Git and GitHub.
> 
> You can continue learning about Git and GitHub by exploring the [GitHub
> Docs](https://docs.github.com/en) where you can learn more about e.g.,
> [repositories](https://docs.github.com/en/repositories) and [GitHub
> Issues](https://docs.github.com/en/issues).
> 
> If you are interested in learning about modern approaches and workflows
> in R (that also touch upon Git and GitHub), take loot at our workshop
> series [r-cubed: Reproducible Research in
> R](https://r-cubed.rostools.org/). As this workshop, the workshop
> material is free and open to everyone.

Reviewer: ```suggestion
new about GitHub and Git. If you want to learn more about Git and
GitHub, check out the [GitHub
Docs](https://docs.github.com/en) where you can learn more about e.g.,
[repositories](https://docs.github.com/en/repositories) and [GitHub
Issues](https://docs.github.com/en/issues). We also have other workshops 
that we regularly run, check out the rostools [landing page](https://github.com/rostools) to learn about them.
```

I'd rather keep it general so that we don't always have to remember to update these sections when things change.

Author: The fact that the plan of the website workshop has already changed, perfectly illustrates your point, haha 🙌
